cpp/src/IceGrid/TraceLevels.h

```diff
@@ -13,37 +13,37 @@ namespace IceGrid
     public:
         TraceLevels(const Ice::CommunicatorPtr&, const std::string&);
 
-        const int admin;
+        const int admin{};
```

Comment:
is this really necessary? shouldn't it be `{0}` like in other places.

Reply:
Fixed as suggested.